raiden/resolver/server.key

```diff
@@ -0,0 +1,28 @@
+-----BEGIN PRIVATE KEY-----
+MIIEvgIBADANBgkqhkiG9w0BAQEFAASCBKgwggSkAgEAAoIBAQDWhS2O4i/XEaPU
+eGG5MDIdP89N3pkJ3XBMxxzMW6ReRNRe4JYKWqTKcGjPaXFV+oMbDquMD/3eA6Iu
+Uw8B7HJfRWSDY1JMUMSPEAxYYSUcXZMVraL4iXUo1V9fH3a5Fm3+PdLchRYCONIF
+6mHOq31mnripTJarwp9zWQEUG9DojB5XAMzX3SNI5XNqFAyoRO2DUpaZ36XeDyjL
+DSBR049jLkHy5JzFz9f8eN/JVivUBlLKVWYTvIRZnU8b8JCnbb/rtYaCIMVT3zvq
+1jDPaZ3byIBJaGbv/p5uXEqi/hoDwRIye+t/B58fb6cAdyVMhTzuEbvFAfcVXlK5
+PNPJuAXbAgMBAAECggEAd0ws388Nx875iwk/X1tcI0P/J1DpTsEk2K3KaCN2ry4b
+5H+FMVDjX+lPgDklXCkqi/uGjye9iKAAqz5plCpIGI9grGipteMczGqiUuwDVVj8
+mQutHAZlb0Xp+d5nidrdZIxBHX2tlaK/uaZ0qhhYbp9KFmyPRgm/3sfaf7JQZWGy
+KmVlqr+eDj2aw/xnVrV917p4Ulo2CbGAknXf51WWYpcjsv/UZ1WmyWEM95PRnqG2
+b7ikN3kKMEiBeFDrw/iipWI7NMfNiGyhp/ZiVMugizCj5zUslO7PSsEagUWDYMNG
+nfgEg3ABIkFI32LAyAHJYnsGMMVsPlO0fGBRjeQcQQKBgQDwXY+LCYQKMVdCsisz
+/O0qOOaAdumaXm/fTI/CmvzvfPHhczCtqL0oKEMz3wCuKcNbaChTXSwRW/fftneu
+bMkrQhouOq1ZfUJgQESVHROdvi8gNL8knRyRQD/9zhaZmayCefa6c3YFvE6UHyVi
+jQ6baL3fCUGYonaQlHuyJnRnMQKBgQDkeUF+J/A1agVIoZtcgPCIby8hjU1Wawt7
++8hEtsknhO1slE0B2H1ImH3ybs2Cl8PLcdw9hGOuFQut9hE/7JagZIL+xWK+bQ20
+4AWWgSrnwBYLlPvV9JFbbnn+HfgA8HJSxWJK4jUPTkr1HxPrYkq4eUHgOAKkISxg
+xmpM0gTSywKBgQCTeqR/R6hnZJzIxG7zqAWQeXccbVmpqLX1iZFPcy8sdjom/Tlp
+4/WjtQh+xB4zvlYqOH89nxZR1F8DOX63W/IWg6lyofgIx205h1VNNJj7mVxq41JA
+UdHcYVKiFJgTSJHrQJAdGIjavg5snaufizLiMeMF/TTtRQmAbkpzJVsYkQKBgEJi
+QPdI33CFp5dc7WlJRZo+w/D5v3jWjXWuyJe+jF/hLaMYy+83Wd+X/zBLpYbGVets
+39M3/kZ8cvJOxuBN+84Kpk0kSPaPdSYS2UzLpw3DMCSZWzrlXr60MNn3FzIeqvkF
+xPOBGm2iEAF80CraWl+TS+K8I3hQ8zYDMKu1chVVAoGBAJASU48KBgyo0MJNXu/o
+SYuzMzcxv0CjJTM8TKYi10/N9ZMPOvO39IxtQjFun+vodhDHRp69fTeF20lfQQir
+Ba8JUB5c/OQMwOOEFC+MQurr85plXJIJWTDgrj+pnWMoU/g+w8rVl0Rd95DvLKNE
+C6dFvsdE4kRdetCoWi79Bpzd
+-----END PRIVATE KEY-----
```

Comment:
It's weird to see a private key here.

Reply:
The resolver folder includes `resolver_server.py` which can be used to simulate a real hash resolver. The communication with the `resolver_server` is done via gRPC over TLS. For that you need a key pair. For testing only, I created a key pair server.crt and server.key that can be used with the `resolver_server`. This will not be used a real environment.